File: Upcoder/Khối 11/Chương 6 Xâu/Xâu dài hơn/K11_XauDaiHon.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    string a, b;
    getline(cin, a);
    getline(cin, b);
    if(a.size() > b.size())
    cout << a;
    else if(a.size() < b.size())
    cout << b;
    else
    cout << "Hai xau dai bang nhau";
    return 0;
}

File: Upcoder/Khối 11/Chương 6 Xâu/Xâu dài hơn/K11_XauDaiHon.cs
using System;
namespace Xaudaihon
{
    public class Program
    {
        static void Main(string[]args)
        {
            string a = Console.ReadLine();
            string b = Console.ReadLine();
            if (a.Length > b.Length)
                Console.WriteLine(a);
            else if (a.Length < b.Length)
                Console.WriteLine(b);
            else
            Console.WriteLine("Hai xau dai bang nhau");
        }
    }
}